clients/bigquery/merge.go

```diff
@@ -227,7 +229,7 @@ func (s *Store) Merge(ctx context.Context, tableData *optimization.TableData) er
 	}
 
 	mergeQuery, err := dml.MergeStatement(ctx, &dml.MergeArgument{
-		FqTableName:               tableData.ToFqName(ctx, constants.BigQuery, true),
+		FqTableName:               fqName,
```

Comment:
In the original code the second argument was `constants.BigQuery` but now it's going to be `s.Label()`.

Reply:
You're right! I wanted to use `s.Label()` since it does the same thing and it consolidates our usage of `constants.BigQuery`, see code: https://github.com/artie-labs/transfer/blob/abb9ca73258b3c014b404f609fe035a137948f5a/clients/bigquery/bigquery.go#L59

Reply:
Ah I hadn't gotten around to looking at the source for `Label()` yet, good call.